process/errors.go

```diff
@@ -451,8 +448,8 @@ var ErrInvalidPeerAccount = errors.New("invalid peer account")
 // ErrInvalidMetaHeader signals that a wrong implementation of HeaderHandler was provided
 var ErrInvalidMetaHeader = errors.New("invalid header provided, expected MetaBlock")
 
-// ErrNilEndOfEpochTrigger signals that a nil end of epoch trigger was provided
-var ErrNilEndOfEpochTrigger = errors.New("nil end of epoch trigger")
+// ErrNilEpochStartTrigger signals that a nil end of epoch trigger was provided
```

Comment:
update comment and error message to match the error

Reply:
done